### ISSUE TITLE
Fuild: How to save file after closing the window widget

### ISSUE THREAD

**Issue by felixmaker** [fltk/fltk#747]
When I close the window widget, it seems the "save" menu item does not activate.

However, I see the source code in code view has changed from visible to hide!

```
Fl_Window window {
      label {Combine Option} open selected
      xywh {321 173 300 157} type Double hide modal
```

**Comment by MatthiasWM** [comment]
Yes, this was originally done because hiding the window in FLUID does not change the generated code. I did update a lot of similar cases, and eventually decided to no longer grey out the "Save" menu in 1.4, fixing all edge cases.

For older version of FLUID, I used to select any widget, and change and unchange the widget order by pressing F2/F3, which activates the Save menu. Not great, but it works.

I will leave this open for now because the "changed" indicator (*) in the main window title should still appear when a window is hidden (or the .fl file is changed in any other way).
